hubblestack/extmods/utils/customUserConf.py

```diff
@@ -0,0 +1,59 @@
+import json
+import yaml
+import logging
+base_path_linux = '/etc/hubble/hubble.d/'
+base_path_windows = 'C:\\Program Files (x86)\\Hubble\\etc\\hubble\\hubble.d\\'
+log = logging.getLogger(__name__)
+default_config_name = 'user.conf'
+
+def parse_configure_option(configure_options):
+    try:
+        keys = configure_options.split(" ")
+        configure_options_dict = {}
+        for str in keys:
+            key = str.split("=")[0]
+            value = str.split("=")[1]
+            configure_options_dict[key] = value
+    except:
+        raise Exception('some error occured, parameters format incorrect')
+    return configure_options_dict
+
+
+def create_user_conf(splunk_conf, configure_options, is_windows):
+    '''
```

Comment:
Could you please use double quotes for your docstrings? We're trying to stay in line with the official guidance, which is double quotes for docstrings.